src/Sylius/Bundle/AdminBundle/Resources/views/Layout/_logo.html.twig

```diff
@@ -1,5 +1,13 @@
-<a class="item" href="{{ path('sylius_admin_dashboard') }}" style="padding: 13px 0;">
-    <div style="max-width: 90px; margin: 0 auto;">
-        <img src="{{ asset('assets/admin/img/admin-logo.svg') }}" class="ui fluid image">
-    </div>
-</a>
+{% if use_webpack %}
+    <a class="item" href="{{ path('sylius_admin_dashboard') }}" style="padding: 13px 0;">
+        <div style="max-width: 90px; margin: 0 auto;">
+            <img src="{{ asset('build/admin/images/admin-logo.svg', 'admin') }}" class="ui fluid image">
+        </div>
+    </a>
+{% else %}
+    <a class="item" href="{{ path('sylius_admin_dashboard') }}" style="padding: 13px 0;">
+        <div style="max-width: 90px; margin: 0 auto;">
+            <img src="{{ asset('assets/admin/img/admin-logo.svg') }}" class="ui fluid image">
```

Comment:
Isn't path change a BC Break? Can't we maintain an old path to simplify migration? At the end, it is a smell for me, if we have to have if statements in templates to make it work

Reply:
I think we discussed it on the morning meeting. The paths are different and most of community using webpack is already using the new path so there's possibility we would introduce more problems changing the path for webpack.